pkg/api/types.go

```diff
@@ -793,6 +793,9 @@ type Container struct {
 	// and shouldn't be used for general purpose containers.
 	Stdin bool `json:"stdin,omitempty"`
 	TTY   bool `json:"tty,omitempty"`
+
+	// Optional: Docker labels
+	Labels map[string]string `json:"labels,omitempty"`
```

Comment:
Do we really want to expose this through the API?  What are you hoping to achieve by exposing it, and how will this map to other runtimes?  We already have pod labels (though even those are not a great fit for Docker labels).